tf/getter.go

```
@@ -174,6 +197,59 @@ func (tfrGetter *RegistryGetter) Get(dstPath string, srcURL *url.URL) error {
+	modulePath = strings.TrimPrefix(modulePath, "/")
+	moduleVersionsPath := fmt.Sprintf("%s/%s/versions", moduleRegistryBasePath, modulePath)
+	moduleVersionsURL, err := url.Parse(moduleVersionsPath)
+	if err != nil {
+		return "", errors.New(err)
+	}
+	bodyData, _, err := httpGETAndGetResponse(ctx, logger, *moduleVersionsURL)
+	if err != nil {
+		return "", errors.New(ModuleVersionsErr{moduleName: modulePath})
+	}
+	var responseJSON Modules
+	if err := json.Unmarshal(bodyData, &responseJSON); err != nil {
```

golangci-lint (1.62.2)

only one cuddle assignment allowed before if statement

(wsl)